core/src/test/java/org/ocpsoft/PrettyTimeI18n_SO_Test.java

```diff
@@ -0,0 +1,508 @@
+
+      final LocalDateTime fiveTicks = now.plusSeconds(25);
+      Assert.assertEquals("isburburi kadib: 5 tik ... BILOW!", t.format(fiveTicks));
+      t.setReference(fiveTicks);
+      Assert.assertEquals("isburburintii waxay ahayd: 5 tik kahor...", t.format(now));
+   }
+
+   @Test
+   public void testYearsAgo() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      Assert.assertEquals("3 sank kahor", t.format(now.minusYears(3)));
+   }
+
+   @Test
+   public void testDecadesAgo() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      Assert.assertEquals("3 toban sanooyin kahor", t.format(now.minus(3, ChronoUnit.DECADES)));
+   }
+
+   @Test
+   public void testCenturiesAgo() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      Assert.assertEquals("3 qarni kahor", t.format(now.minus(3, ChronoUnit.CENTURIES)));
+   }
+
+   @Test
+   public void testWithinTwoHoursRounding() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      Assert.assertEquals("2 saac kahor", t.format(now.minusHours(1).minusMinutes(45)));
+   }
+
+   @Test
+   public void testPreciseInTheFuture() throws Exception
+   {
+      PrettyTime t = new PrettyTime();
+      List<Duration> durations = t.calculatePreciseDuration(now.plusHours(5).plusMinutes(10).plusSeconds(1));
+      Assert.assertTrue(durations.size() >= 2);
+      Assert.assertEquals(5, durations.get(0).getQuantity());
+      Assert.assertEquals(10, durations.get(1).getQuantity());
+   }
+
+   @Test
+   public void testPreciseInThePast() throws Exception
+   {
+      PrettyTime t = new PrettyTime();
+      List<Duration> durations = t.calculatePreciseDuration(now.minusHours(5).minusMinutes(10).minusSeconds(1));
+      Assert.assertTrue(durations.size() >= 2);
+      Assert.assertEquals(-5, durations.get(0).getQuantity());
+      Assert.assertEquals(-10, durations.get(1).getQuantity());
+   }
+
+   @Test
+   public void testFormattingDurationListInThePast() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      List<Duration> durations = t.calculatePreciseDuration(now.minusDays(3).minusHours(15).minusMinutes(38));
+      Assert.assertEquals("3 maalmood 15 saac 38 daqiiqo kahor", t.format(durations));
+   }
+
+   @Test
+   public void testFormattingDurationListInTheFuture() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      List<Duration> durations = t.calculatePreciseDuration(now.plusDays(3).plusHours(15).plusMinutes(38));
+      Assert.assertEquals("3 maalmood 15 saac 38 daqiiqo kadib", t.format(durations));
+   }
+
+   @Test
+   public void testSetLocale() throws Exception
+   {
+      PrettyTime t = new PrettyTime(now);
+      final LocalDateTime threeDecadesAgo = now.minus(3, ChronoUnit.DECADES);
+      Assert.assertEquals("3 toban sanooyin kadib", t.format(threeDecadesAgo));
+      t.setLocale(Locale.GERMAN);
+      Assert.assertEquals("vor 3 Jahrzehnten", t.format(threeDecadesAgo));
```

Comment:
Out of curiosity, why are these German assertions here? I don't think these should be in the SO tests. They should already be tested in the _DE locale test files. If there's no specific reason for this other than verifying that locale changing works, could you remove them? Thanks!